rpa-builder/6.2/modules/ROOT/pages/toolbox-measurement-points-stopwatch-collection.adoc

```diff
@@ -0,0 +1,53 @@
+= Stopwatch Collection
+
+The Stopwatch Collection Action Step enables you to collect all timer values from the Stopwatch Action Steps configure in an Activity Workflow. 

```

Comment:
We should mention here that the Stopwatch Collection contains both Timers from the Stopwatch Action Step AND the General Timer Actions Step